rpmlint/checks/ExecDocCheck.py

```diff
@@ -0,0 +1,50 @@
+from pathlib import Path
+import stat
+
+from rpmlint.checks.AbstractCheck import AbstractCheck
+
+
+class ExecDocCheck(AbstractCheck):
+    @staticmethod
+    def ignore_pkg(name):
+        if name.startswith('bundle-') or '-devel' in name or '-doc' in name:
+            return True
+        return False
+
+    def check(self, pkg):
+        if pkg.isSource():
+            return
+
+        self._check_executable_docs(pkg)
+        if not self.ignore_pkg(pkg.name):
+            self._check_huge_docs(pkg)
+
+    def _check_executable_docs(self, pkg):
+        files = pkg.files()
+        for f in pkg.docFiles():
+            mode = files[f].mode
+            if not stat.S_ISREG(mode) or not mode & 0o111:
+                continue
+            path = Path(f)
+            extensions = ['.txt', '.gif', '.jpg', '.html', '.pdf', '.ps', '.pdf.gz', '.ps.gz']
+            if path.suffix in extensions:
+                self.output.add_info('E', pkg, 'executable-docs', f)
+
+            for name in ['README', 'NEWS', 'COPYING', 'AUTHORS']:
```

Comment:
Maybe consider LICENSE, and LICENCE and all the variants we have in the list case insensitive? It is not a bug per-se but I think looking on the current spec files it is quite used...